fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRecordCursor.java

```diff
@@ -232,14 +243,24 @@ private synchronized IndexReader getIndexReader() throws IOException {
     private void performScan() throws IOException {
```

Comment:
This needs some instrumentation:
* `Event` for calling `searcher`.
* `Count` for number of docs returned by that.